.github/workflows/ci.yaml

```diff
@@ -52,20 +52,30 @@ jobs:
         name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
       -
-        name: Login to Registry
+        name: Login to GitHub Container Registry
         uses: docker/login-action@v1 
         with:
-          registry: docker.pkg.github.com
-          username: ${{ secrets.USERNAME }}
-          password: ${{ secrets.TOKEN }}
+          registry: ghcr.io
+          username: ${{ secrets.GH_USERNAME }}
+          password: ${{ secrets.GH_PAT }}
+      -
+        name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
       -
         name: Build and push
         id: docker_build
         uses: docker/build-push-action@v2
         with:
           platforms: linux/amd64,linux/arm64,linux/arm
           push: true
-          tags: docker.pkg.github.com/argoproj-labs/argocd-notifications/argocd-notifications:${{ env.IMAGE_TAG }}
+          tags: |
+            argoproj-labs/argocd-notifications:${{ env.IMAGE_TAG }}
```

Comment:
Can we avoid pushing non latest tag to `argoproj-labs` ? 

We are trying to use dockerhub/argoproj-labs only for official releases. I was hoping to automate it using separate GH action workflow that is triggered manually (similar to https://github.com/argoproj/argo-cd/blob/master/.github/workflows/release.yaml)

Reply:
@alexmt Would a model similar to this work: https://github.com/argoproj/argo-cd/blob/master/.github/workflows/release.yaml#L2-L12 were only pushes with specific tags cause a push to DockerHub? or do you want it totally manual "click a UI button"?

Additionally, when pushing to DH if we choose the above model where it pushed actual tagged releases _and_ the `latest` tag - is that acceptable, or are you asking for it to only ever push one image tagged `latest` and ignore all of the `v1.x.x` tags?

Reply:
I was thinking about Argo CD model - it uses tag with a special name as trigger. After I've learned about `workflow_dispatch` from your previous PR and now hoping to use Github UI instead. I'm asking to push only to 'latest' on every push to master.

> I can set up two actions workflows then, one for CI for general PRs and merges like we currently have, and another for tagged branches that are actual "releases" if that's what you're after.

Very appreciate your help here! 

If you don't mind can you remove `vx.x.x` tag pushes in this PR ( just want to get CI green ) and work on release workflow in next PR?

Reply:
Works for me, let me open another PR for that!

Reply:
Fixed @alexmt, will now only push `latest` :)

Reply:
@alexmt on the failure - you need to provide the repo secrets:

`GH_USERNAME` - a user with ownership over the argoproj-labs GH Org
`GH_PAT` - a personal access token with package:read, package:write and package:delete permissions. more here: https://docs.github.com/en/packages/guides/migrating-to-github-container-registry-for-docker-images#authenticating-with-the-container-registry

Reply:
thank you! doing it now

Reply:
all green https://github.com/argoproj-labs/argocd-notifications/runs/2034692703 !